Sources/CCFilter/ViewControllers/CCRootFilterViewController.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol CCRootFilterViewControllerDelegate: CCViewControllerDelegate {
```

Comment:
Is inheriting another protocol really necessary? Can't you maybe specify that rootDelegate should confirm to both instead?